test/FunctionalTests/Microsoft.OData.Core.Tests/Build.NetFramework/Microsoft.OData.Core.Tests.csproj

```diff
@@ -157,7 +157,7 @@
     <Compile Include="..\JsonLight\ODataJsonLightUtilsTests.cs" />
     <Compile Include="..\JsonLight\ODataJsonLightValidationUtilsTests.cs" />
     <Compile Include="..\JsonLight\ODataJsonLightValueSerializerTests.cs" />
-    <Compile Include="..\JsonLight\PayloadValueConverters.cs" />
+    <Compile Include="..\JsonLight\ODataSimplifiedOptionsTests.cs" />    <Compile Include="..\JsonLight\PayloadValueConverters.cs" />

```

Comment:
><Compile Include="..\JsonLight\PayloadValueConverters.cs" /> [](start = 73, length = 60)

 accident?

Reply:
yes; thanks.

---
In reply to: [279947483](https://github.com/OData/odata.net/pull/1454#discussion_r279947483) [](ancestors = 279947483)